frontend/src/scenes/dashboard/EmptyDashboardComponent.tsx

```diff
@@ -95,6 +95,7 @@ export function EmptyDashboardComponent({ loading }: { loading: boolean }): JSX.
                                 icon={<IconPlus />}
                                 center
                                 fullWidth
+                                disabled={!canEdit}
```

Comment:
Sorry I didn't leave a review, when I saw this I was like – this is going to be confusing for users. Hence I added `disabledReason` to `LemonButton`: https://github.com/PostHog/posthog/pull/13690 😄 I propose we switch to that here as soon as that PR is in.

Reply:
@liyiy If you've got any downtime, feel free to to make a PR switching to `disabledReason`, otherwise we may migrate this… at some point, not sure though

Reply:
yepp can do 😺 was just waiting for that other PR to merge